### PR TITLE
Update my staffmon

### PR DESCRIPTION
**Pull request by ghost**
Me and insist talked about it. Item takes away HP and Special move give the user 25% andhas the same use rate as protect so it discourages double useage. It should be balanced unless insist disagrees

### CODE REVIEW

**Comment by ghost** [comment]
Fair enough insist

**Comment by DeathlyPlays** [comment]
hmm lemme review, btw make a ssb-data.txt entry

File: mods/essb/abilities.js [hunk 1]
```diff
@@ -554,25 +554,23 @@ exports.BattleAbilities = {
...
-		onModifyMove: function (move) {
-			move.ignoreAbility = true;
-		},
-		onSourceModifyDamage: function (damage, source, target, move) {
-			if (this.random(10) < 5) {
-				return this.chainModify(false);
+	"sysop": {
+		id: "sysop",
+		name: "Sys-OP",
+		onPrepareHit: function (source, target, move) {
```
Comment (DeathlyPlays) [inline mods/essb/abilities.js:560]: If this has Wonder Guard AND Protean it'll keep changing type and players won't be able to KO it....

Comment (DeathlyPlays) [reply mods/essb/abilities.js:560]: ^

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

File: mods/essb/abilities.js (hunk 1 above)
Comment (ghost) [reply mods/essb/abilities.js:560]: True

Comment (ghost) [reply mods/essb/abilities.js:560]: But then the curse should Ko it

Comment (ghost) [reply mods/essb/abilities.js:560]: Either way

**Comment by ghost** [comment]
@BamdTheClueless thoughts? Does my Pichu need a nerf?

**Comment by DeathlyPlays** [comment]
Yes it does....

File: mods/essb/items.js [hunk 2]
```diff
@@ -121,4 +121,31 @@ exports.BattleItems = {
...
+		onModifySpDPriority: 2,
+		onModifySpD: function (spd, pokemon) {
+			if (pokemon.baseTemplate.nfe) {
+				return this.chainModify(1.5);
+			}
+		},
+		onResidualOrder: 26,
+		onResidualSubOrder: 2,
+		onResidual: function (pokemon) {
+			this.damage(pokemon.maxhp / 4);
```
Comment (ghost) [inline mods/essb/items.js:146]: This should justify the mon's powers and abilities

Comment (ghost) [reply mods/essb/items.js:146]: The curse effect is: A cursed Pokémon will lose ¼ of its maximum HP at the end of each turn.

Comment (DeathlyPlays) [reply mods/essb/items.js:146]: Oh yeah...

**Review by DeathlyPlays — APPROVED** [review]

**Comment by ghost** [comment]
Plus we can buff and debuff if necessary

**Pull request merged**